### PR TITLE
Text changes to Home, General Guidance, Relationships, Use Cases, Profiles (FHIR-46803, FHIR-46809, FHIR-46810, FHIR-46811, FHIR-46812)

### PR DESCRIPTION
**Pull request by JayMurdoch**
Fix for [FHIR-46803](https://jira.hl7.org/browse/FHIR-46803)
* Add note to Home > 1.5 AU eRequesting FHIR RESTful interactions.

Fix for [FHIR-46809](https://jira.hl7.org/browse/FHIR-46809) 
* Remove General Guidance (contained only Read/Search Syntax) and changed references in profiles to AU Core Read/Search Syntax.

Fix for [FHIR-46810](https://jira.hl7.org/browse/FHIR-46810)
* Remove Figure 1 and preceding text. 
* Update Figure 2 as described.

[FHIR-46811](https://jira.hl7.org/browse/FHIR-46811)
* Update Use Case text as described.
* Changed page name/links from 'AU eRequesting Data for Interoperability' to 'AU eRequesting Data for Interoperability Mappings'

[FHIR-46812](https://jira.hl7.org/browse/FHIR-46812)
* Update Extensions section in Profiles and Extensions page to reflect referenced extensions.

### CODE REVIEW

File: input/pagecontent/index.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ The diagram below illustrates the interactions between the various actors:
 *Figure 1: FHIR RESTful interactions*
 <br/>
 
+*Note*: AU eRequesting Actor interactions will transition with the introduction of the Task resource, which will become central to fulfilment.
```
Comment (dbojicic) [inline input/pagecontent/index.md:59]: I don't think this wording works anymore as the Task profile is included in the IG now.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments.

File: input/pagecontent/index.md (hunk 1 above)
Comment (dbojicic) [reply input/pagecontent/index.md:59]: The wording has been confirmed with @michael-wilson-au, resolving this comment.

**Review by dbojicic — APPROVED** [review]

File: input/pagecontent/index.md (hunk 1 above)
Comment (JayMurdoch) [reply input/pagecontent/index.md:59]: Confirmed with @michael-wilson-au that wording remains appropriate as review of interactions has not yet taken place.

File: input/pagecontent/profiles-and-extensions.md [hunk 2]
```diff
@@ -11,13 +11,18 @@ The following is the full list of profiles that have been defined for this imple
...
+
+* [Australian Indigenous Status](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-indigenous-status.html) in [AU Core Patient](http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient)
+* [Individual Pronouns](http://hl7.org/fhir/StructureDefinition/individual-pronouns) in [AU Core Patient](http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient)
+* [Individual Gender Identity](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) in [AU Core Patient](http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient)
 
 <!-- ================================================ -->
 <!--  use this line to include an autogenerated list of all profiles and highlight new ones using the input/data/new_stuff.yml list.  Remove it if you would like to hand generate it -->
 
-{% include ext-list-generator.md %}
+<!-- {% include ext-list-generator.md %}
```
Comment (michael-wilson-au) [inline input/pagecontent/profiles-and-extensions.md:25]: Is there missing close tag on this comment?

**Review by michael-wilson-au — CHANGES_REQUESTED** [review]

File: input/pagecontent/profiles-and-extensions.md (hunk 2 above)
Comment (dbojicic) [reply input/pagecontent/profiles-and-extensions.md:25]: Yes, there is! It still renders fine but we should fix it.

**Review by dbojicic — CHANGES_REQUESTED** [review]
@JayMurdoch, just one more small change.

File: input/pagecontent/profiles-and-extensions.md (hunk 2 above)
Comment (JayMurdoch) [reply input/pagecontent/profiles-and-extensions.md]: Including close tag caused it to render in IG - possibly due to issue in template. 
Content deleted as currently not required, and can be added back as needed.

**Review by dbojicic — APPROVED** [review]

**Review by michael-wilson-au — APPROVED** [review]

**Pull request merged**